Lib/inspect.py

```diff
@@ -32,7 +32,20 @@
 
 __author__ = ('Ka-Ping Yee <ping@lfw.org>',
               'Yury Selivanov <yselivanov@sprymix.com>')
-
+__all__ = ['classify_class_attrs', 'cleandoc', 'currentframe', 'findsource',
```

Comment:
I'm not a fan of this indention style. Please use

```suggestion
__all__ = [
    'classify_class_attrs', 'cleandoc', 'currentframe', 'findsource',
    ...
]
```